doc/user/bgp.rst

```diff
@@ -1137,6 +1137,12 @@ Peer Filtering
    on reflected routes. This option allows the modifications to be reflected as
    well. Once enabled, it affects all reflected routes.
 
+.. index:: neighbor PEER sender-as-path-loop-detection
```

Comment:
we should add this command for `[no]` as well

Reply:
and would be good having defined what is the default value

Reply:
Fixed.